configure.ac

```diff
@@ -75,7 +75,7 @@ AM_PATH_GLIB_2_0(,,AC_MSG_ERROR([GLib not found]))
 dnl When bumping the gio-unix-2.0 dependency (or glib-2.0 in general),
 dnl remember to bump GLIB_VERSION_MIN_REQUIRED and
 dnl GLIB_VERSION_MAX_ALLOWED in Makefile.am
-GIO_DEPENDENCY="gio-unix-2.0 >= 2.40.0"
+GIO_DEPENDENCY="gio-unix-2.0 >= 2.50.0"
```

Comment:
I’ve factored it out twice: e03fd8f bumps the dependency to 2.44, which is what it should be on `master`, since `g_autoptr()` requires that. 4e288b0 increases the maximum dependency to 2.50, which is needed for `g_drive_is_removable()`. If 2.50 is not available, the `g_drive_is_removable()` call is dropped, which hits performance but not functionality (technically it does, but in reality it’s not a problem).